oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/query/ElasticRequestHandler.java

```diff
@@ -578,10 +588,70 @@ private boolean visitTerm(String propertyName, String text, String boost, boolea
                 return true;
             }
         });
-        
+
         return Query.of(q -> q.bool(result.get()));
     }
 
+    private ObjectBuilder<BoolQuery> inference(BoolQuery.Builder b, String propertyName, String text, PlanResult pr, boolean dbEnabled) {
+        ElasticIndexDefinition.InferenceDefinition.Query q = null;
```

Comment:
Could you write a comment that just briefly explains when the inference query could be null and what happens when that is the case? Looks like it will just do a basic fulltext search, but am not 100% sure.

Reply:
added comment